src/test/java/api/requests/RequestsAPICreationTests.java

```diff
@@ -1286,6 +1286,66 @@ public void cannotCreateHoldRequestWhenItemIsAvailable()
       hasParameter("requestType", "Hold"))));
   }
 
+  @Test
+  public void cannotCreateTwoRequestsFromTheSameUser()
+    throws InterruptedException,
+    ExecutionException,
+    TimeoutException,
+    MalformedURLException {
+
+    final IndividualResource checkedOutItem = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource requestPickupServicePoint = servicePointsFixture.cd1();
+    final IndividualResource jessica = usersFixture.jessica();
+
+    loansFixture.checkOut(checkedOutItem, jessica);
+
+    requestsClient.create(new RequestBuilder()
+      .recall()
+      .forItem(checkedOutItem)
+      .withPickupServicePointId(requestPickupServicePoint.getId())
+      .by(jessica));
+
+    final Response response = requestsClient.attemptCreate(new RequestBuilder()
+      .recall()
```

Comment:
Is there another validation check for multiple recall requests for the same item?

Reply:
I don't think so